packages/editor/src/components/post-url/style.scss

```diff
@@ -27,6 +27,12 @@
 	padding-inline-start: 0 !important;
 }
 
-.editor-post-url__panel-toggle {
-	word-break: break-word;
+.editor-post-panel__row-control {
+	.components-button.editor-post-url__panel-toggle {
```

Comment:
Let's avoid using `.components-*` classnames, as they are not assumed to be public APIs from the `@wordpress/components` package.

Reply:
@ciampo Thanks for the review. Can you please review it again?